src/main/java/seedu/anilist/storage/JsonSerializableAnimeList.java

```diff
@@ -17,26 +17,26 @@
  * An Immutable AniList that is serializable to JSON format.
  */
 @JsonRootName(value = "anilist")
```

Comment:
anilist here and on line 17

Reply:
Changed line 17, but for line 19 I think that refers to the package name. which will be 'anilist'